extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/src/main/java/io/quarkus/opentelemetry/exporter/otlp/runtime/graal/OtlpExporterSubstitutions.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.opentelemetry.exporter.otlp.runtime.graal;
+
+import java.io.ByteArrayInputStream;
+import java.net.URI;
+
+import javax.net.ssl.SSLException;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.Metadata;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import io.grpc.stub.MetadataUtils;
+
+@TargetClass(className = "io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporterBuilder")
+final class Target_io_opentelemetry_exporter_otlp_trace_OtlpGrpcSpanExporterBuilder {
```

Comment:
I think it would be nice to have a class level comment explaining why this substitution.